docs/man/docker-create.1.md

```diff
@@ -102,6 +104,12 @@ IMAGE [COMMAND] [ARG...]
                                'container:<name|id>': reuses another container shared memory, semaphores and message queues
                                'host': use the host shared memory,semaphores and message queues inside the container.  Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
 
+**-l**, **--label**=[]
+   Set metadata on the container (e.g., --label=com.example.key=value)
```

Comment:
I don't know if set or add is the right verb.  The behavior is similar to ENV.  The image has labels which are copied to the container and then `-l` can add additional ones.  So maybe `add` is the right term.